### ISSUE TITLE
Bring character and journal tags up to par with submissions

### ISSUE THREAD

**Issue by taedixon** [Weasyl/libweasyl#2]
Update the schema to allow storing tag history for journals and characters
This is a dependency of Weasyl/weasyl#11

**Comment by weykent** [comment]
Or maybe hopefully instead finally realize the plan to collapse them all into submissions?

**Comment by skylerbunny** [comment]
One of these has immediate (positive) consequences to our userbase and involves putting code we already use in another location, which should be pretty easy. The other, while we want to do it, is more difficult. I'm not saying we don't WANT to do it, but we get active complaints from our userbase now when tag colorization is mentioned in discussion and it's realized after the fact 'oh right it's a journal, it doesn't do that yet'.

tl;dr - I agree we want to merge them, but it isn't a prerequisite to this easier task.

**Comment by weykent** [comment]
Exactly: it means duplicating code, and duplicating maintenance burden.

**Comment by weykent** [comment]
Moved to Weasyl/weasyl#44.
